plugin/storage/integration/kafka_test.go

```diff
@@ -48,7 +48,7 @@ func (s *KafkaIntegrationTestSuite) initialize() error {
 	topic := "jaeger-kafka-integration-test-" + strconv.FormatInt(time.Now().UnixNano(), 10)
 
 	f := kafka.NewFactory()
-	v, command := config.Viperize(f.AddFlags, app.AddFlags)
+	v, command := config.Viperize(app.AddFlags)
```

Comment:
@yurishkuro the change here fixed the test. `app.AddFlags` already contained all flags needed by `f.InitFromViper(v)`

Reply:
is kafka factory's AddFlags() no-op then?looks a bit odd